migration/v2.19.0-ck8s1-v2.19.0-ck8s2/upgrade-cluster.md

```diff
@@ -59,4 +59,18 @@
 
 ## Update terraform state for Openstack environments
 
-1. If you're running on an openstack cloud provider, you will have to update the terraform state. This can be done by exporting `CK8S_CONFIG_PATH`, `OS_USERNAME` and `OS_PASSWORD`, and then running `migrate-terraform-openstack.sh` and following the instructions that are shown after the script finishes.
+If you're running on an Openstack cloud provider, you will have to update the terraform state. *This only applies to Openstack environments that have been upgraded from kubespray v2.18. For environments that were initially set up on v2.19, you do not need to perform the steps below.*
+
+1. For Safespring environments only, add the following to your `sc-config/cluster.tfvars` and `wc-config/cluster.tfvars`:
```

Comment:
Do we need to include this in the templates? to have them by default in the new safespring environments?

Reply:
> Do we need to include this in the templates? to have them by default in the new safespring environments?

It is already in the safespring template